roles/load-balancers/manage-haproxy/handlers/main.yml

```diff
@@ -7,21 +7,30 @@
     state: started
   with_items:
   - haproxy
+  become: True
 
 - name: 'restart rsyslog'
   service:
     name: rsyslog
     state: restarted
+  become: True
 
 - name: 'reload haproxy'
-  service: 
+  service:
     name: haproxy
-    state: reloaded 
+    state: reloaded
+  become: True
 
 - name: 'remove tmp new file'
   file:
     path: '{{ temp_new_file }}'
     state: absent
+  delegate_to: localhost
+  run_once: True
 
```

Comment:
We need to add `become: True` here as well otherwise we'll get another permission denied error.

Reply:
As an FYI, here's the error:

```
RUNNING HANDLER [load-balancers/manage-haproxy : remove tmp new file] *****************************************************************
fatal: [localhost -> localhost]: FAILED! => {"changed": false, "gid": 0, "group": "root", "mode": "0644", "msg": "unlinking failed: [Errno 13] Permission denied: '/etc/haproxy/haproxy.cfg.new' ", "owner": "root", "path": "/etc/haproxy/haproxy.cfg.new", "secontext": "system_u:object_r:etc_t:s0", "size": 2176, "state": "file", "uid": 0}
```

And here are the permissions:

`-rw-r--r--. 1 root root 2176 Jul 17 21:59 /etc/haproxy/haproxy.cfg.new`

Reply:
thx - fixed